showcase/app/templates/components/advanced-table.hbs

```diff
@@ -0,0 +1,1710 @@
+        {{#each booleans as |bool2|}}
+          <SF.Item @label="{{if bool1 'Selected' 'Deselected'}} {{if bool2 ' with sort'}}">
+
+            <div class="hds-advanced-table hds-advanced-table--density-medium" {{style gridTemplateColumns="auto 1fr"}}>
+              <div class="hds-advanced-table__thead">
+                <div class="hds-advanced-table__tr">
+                  <Hds::AdvancedTable::ThSelectable
+                    @selectionScope="col"
+                    @isSelected={{bool1}}
+                    @onClickSortBySelected={{if bool2 this.noop}}
+                  />
+                  <Hds::AdvancedTable::Th>Lorem</Hds::AdvancedTable::Th>
+                </div>
+              </div>
+              <div class="hds-advanced-table__tbody">
+                <div class="hds-advanced-table__tr">
+                  <Hds::AdvancedTable::ThSelectable @selectionScope="row" @isSelected={{bool1}} />
+                  <Hds::AdvancedTable::Td>Ipsum</Hds::AdvancedTable::Td>
+                </div>
+              </div>
+            </div>
+            <br />
+
+            <div class="hds-advanced-table hds-advanced-table--density-medium" {{style gridTemplateColumns="auto 1fr"}}>
+              <div class="hds-advanced-table__thead">
+                <div class="hds-advanced-table__tr">
+                  <Hds::AdvancedTable::ThSelectable
+                    @selectionScope="col"
+                    @isSelected={{bool1}}
+                    @onClickSortBySelected={{if bool2 this.noop}}
+                    mock-state-value="focus"
+                    mock-state-selector="input"
+                  />
+                  <Hds::AdvancedTable::Th>Lorem</Hds::AdvancedTable::Th>
+                </div>
+              </div>
+              <div class="hds-advanced-table__tbody">
+                <div class="hds-advanced-table__tr">
+                  <Hds::AdvancedTable::ThSelectable
+                    @selectionScope="row"
+                    @isSelected={{bool1}}
+                    mock-state-value="focus"
+                    mock-state-selector="input"
+                  />
+                  <Hds::AdvancedTable::Td>Ipsum</Hds::AdvancedTable::Td>
+                </div>
+              </div>
+            </div>
+          </SF.Item>
+        {{/each}}
+      {{/each}}
+    {{/let}}
+    {{#let (array false true) as |booleans|}}
+      {{#each booleans as |bool|}}
+        <SF.Item @label="Indeterminate {{if bool ' with sort'}}">
+
+          <div class="hds-advanced-table hds-advanced-table--density-medium" {{style gridTemplateColumns="auto 1fr"}}>
+            <div class="hds-advanced-table__thead">
+              <div class="hds-advanced-table__tr">
+                <Hds::AdvancedTable::ThSelectable
+                  @selectionScope="col"
+                  @isSelected={{true}}
+                  @onClickSortBySelected={{if bool this.noop}}
+                  @didInsert={{this.mockIndeterminateState}}
```

Comment:
Question: if you look at the preview for this, the indeterminate state example is broken. does anyone have any thoughts as to what could be going on? 

<img width="333" alt="Screenshot 2024-12-18 at 5 11 33 PM" src="https://github.com/user-attachments/assets/c7aef846-9e33-461f-b78e-807e9cd98111" />